src/Region.h

```diff
@@ -54,6 +54,7 @@ class Region {
   Region* _parent_region;
 
 public:
+  Region();
```

Comment:
Region(regionType region_type = INTERSECTION) would be more flexiable for the user.

Reply:
Only if we create a SWIG typemap so that the user can create those from Python
If I can figure it out quickly enough I ll add it